drivers/fuzz/src/fuzz_target_lms.rs

```diff
@@ -0,0 +1,83 @@
+// Licensed under the Apache-2.0 license
+
+#![cfg_attr(feature = "libfuzzer-sys", no_main)]
+
+#[cfg(all(not(feature = "libfuzzer-sys"), not(feature = "afl")))]
+compile_error!("Either feature \"libfuzzer-sys\" or \"afl\" must be enabled!");
+
+#[cfg(feature = "libfuzzer-sys")]
+use libfuzzer_sys::fuzz_target;
+
+#[cfg(feature = "afl")]
+use afl::fuzz;
+
+#[cfg(not(feature = "struct-aware"))]
+use std::mem::size_of;
+
+use caliptra_drivers::{Lms, Sha256SoftwareDriver};
+use caliptra_lms_types::{LmsPublicKey, LmsSignature};
+
+pub const SHA256_DIGEST_WORD_SIZE: usize = 8;
+// TODO: When using SHA256_DIGEST_WORD_SIZE, these tend to accompany?
```

Comment:
I wonder what happens if we break this expectation. It will make it too difficult to provide a seed corpus though.

Reply:
Since these are specified at compile time, I think it's fine to not fuzz different combinations. But for consistency with what ROM is doing, I think we should depend directly on these P and H values: https://github.com/chipsalliance/caliptra-sw/blob/8cb6e40db5a79148fe2956d78380b46198174848/image/types/src/lib.rs#L37

Reply:
Okay, done. I'll be remaking the seed corpus soon too